ports/espressif/common-hal/socketpool/Socket.c

```diff
@@ -369,7 +369,7 @@ void common_hal_socketpool_socket_connect(socketpool_socket_obj_t *self,
     struct addrinfo *result_i;
     int error = lwip_getaddrinfo(host, NULL, &hints, &result_i);
     if (error != 0 || result_i == NULL) {
-        mp_raise_OSError(EHOSTUNREACH);
+        common_hal_socketpool_socketpool_raise_gaierror(SOCKETPOOL_EAI_NONAME, MP_QSTR_Name_space_or_space_service_space_not_space_known);
```

Comment:
Just to be clear, the gaierror messages are not translatable?

Reply:
I briefly looked but wasn't sure how to use a translatable string here. (It was this way in the first commit too)

I can try to make it translatable if you like.

Reply:
Well, the code to do that is in `py/objexcept.c` and maybe a little in `py/mpprint.c`, but it's not factored nicely. We could make a utility routine to decompress a compressed string into a Python string. I can make an issue for that for later.